docs/INSTALL.md

```diff
@@ -9,23 +9,27 @@ General ZAC usage instructions may be found in the [README.md](../README.md) fil
 
 - Java JDK 17 and up (we use the `Temurin` distribution)
 
+For mac users you can run the following command to install the required Java version:
+
+```shell
+brew install oracle-jdk17
```

Comment:
ho, wacht even. Waarom die Oracle JDK? Die gebruiken we liever niet. De Temurin distro is beter: https://whichjdk.com/. En ook: ik denk niet dat we in onze docs moeten gaan uitleggen hoe je een JDK installeert. Niet iedereen kan of wil Brew gebruiken en daarnaast gebruik je meestal ook nog een tool als Jenv om JDKs te switchen maar ook dat moeten we niet willen documenteren.

Tenslotte is het helemaal niet meer nodig om JDK 17 te gebruiken. Je kan nu ook gewoon de laatste JDK gebruiken (20) en dat werkt ook prima. Heb ik gefixt toen we overgingen naar Gradle.

Reply:
Bij mij werkt het niet als ik jdk 20 gebruik. Er wordt geen artifact gemaakt die ik kan selecteren in wildfly. Toen ik builde met jdk17 werkte het wel.

Reply:
Ik zal dit stukje verwijderen, maar Joran en ik moeten wel nog even naar oplossing zoeken voor jdk-7